extensions/amp-mraid/0.1/amp-mraid.js

```diff
@@ -87,7 +87,16 @@ export class MraidInitializer {
 
     // It looks like we're initiating a network load for mraid from a relative
     // url, but this will actually be intercepted by the mobile app SDK and
-    // handled locally.
+    // handled locally.  To be safe, first we inject an invalid base tag which
+    // prevents any real relative loads.  This works because post-transformation
+    // creatives won't have <base>, and so we're injecting the first <base> tag
+    // on the page.
+    const head = document.getElementsByTagName('head').item(0);
```

Comment:
Please use `this.win.document`. 
Does `<amp-mraid>` works in A4A ads on AMP pages?

Reply:
> Please use this.win.document.

Done 3bf9e7494

>  Does <amp-mraid> works in A4A ads on AMP pages?

It wouldn't work, and is explicitly disabled: `if (getMode().runtime !== 'inabox') { return; }`

Though that's wrong: I need to be calling `HostServices.reject*ServiceForDoc`.  I'll add that in a separate PR.  We want including amp-mraid when serving to A4A on AMP pages to be a no-op, and as-is it will keep a bunch of things from working because they'll wait for it to run.

Reply:
https://github.com/ampproject/amphtml/pull/21728